mention/src/main/proto/spine_examples/pingh/mention/github_client.proto

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+syntax = "proto3";
+
+package spine_examples.pingh.mention;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.pingh.spine.io";
+option java_package = "io.spine.examples.pingh.mention";
+option java_outer_classname = "GitHubClientProto";
+option java_multiple_files = true;
+
+import "spine_examples/pingh/identifiers.proto";
+import "google/protobuf/timestamp.proto";
+
+// The process of update mentions from GitHub client.
+message GitHubClient {
+  option (entity) = { kind: PROCESS_MANAGER };
+
+  // The ID of the GitHub client.
+  GitHubClientId id = 1;
+
+  // The login for the user for whom the update is performed.
+  string login = 2 [(required) = true];
+
+  // The token for the user for whom the update is performed.
+  string token = 3 [(required) = true];
+
+  // Time when the update of mentions started.
+  google.protobuf.Timestamp time = 4 [(required) = true];
```

Comment:
I also don't understand how this field may be required.

Please have a documentation section on the `message` level describing the lifecycle. In particular, what happens once the process ends. 

And how an ID is assigned? Do we auto-generate it? If yes, then how we will be able to tell the update process for a certain user is currently underway?

All these questions should be reflected in the documentation.